src/metabase/api/card.clj

```diff
@@ -702,10 +702,12 @@
 
   `parameters` should be passed as query parameter encoded as a serialized JSON string (this is because this endpoint
   is normally used to power 'Download Results' buttons that use HTML `form` actions)."
-  [card-id export-format :as {{:keys [parameters format_rows]} :params}]
+  [card-id export-format :as {{:keys [parameters]} :params
+                              {:keys [pivot-results format-rows]} :body}]
```

Comment:
I thought that passing format-rows (and now also pivot-results) as body params is a bit better than passing them as query params